src/main/io/ledstrip.h

```diff
@@ -45,7 +45,7 @@
 
 #define LED_POS_BITCNT                  8
 #define LED_FUNCTION_BITCNT             4
-#define LED_OVERLAY_BITCNT              6
+#define LED_OVERLAY_BITCNT              7
```

Comment:
This won't work. You can't just increase the size of one of the flag sets stuffed into the `uint32_t`, and not increment all of the subsequent `_OFFSET_` defines above.

Also, note that there is no space left in the `uint32_t`, so the only way to add this as a new mode is to remove one of the existing modes.

Also, since this will modify the interpretation of the LED config, the version number of the `ledStripConfig` parameter group will have to be incremented in `ledstrip.c`.

Reply:
Or implement it as led function, not overlay ...

Also some checks for this type of error would be nice ...